### ISSUE TITLE
The sheet is not working anymore with the last dnd system update on foundry

### ISSUE THREAD

**Issue by Braste** [jopeek/fvtt-dndbeyond-character-sheet#36]
What the title says, it is not usable anymore.

**Comment by jopeek** [comment]
Yeah I tried making it work but couldn’t. This may be the end of the road for now. Once other people make theirs work, I may check it out to see. I couldn’t make the item entries clickable to actually roll/use/etc the items and spells etc.
